WordPress/Classes/ViewRelated/Notifications/Style/WPStyleGuide+Notifications.swift

```diff
@@ -229,9 +226,6 @@ extension WPStyleGuide {
         fileprivate static let snippetParagraph         = NSMutableParagraphStyle(
             minLineHeight: snippetLineSize, maxLineHeight: snippetLineSize, lineBreakMode: .byWordWrapping, alignment: .natural
```

Comment:
Mmmmmm can't recall *why* we've got WordWrapping here, instead of Truncate Tail. I _suspect_ it may be related to the width constraints (Snippet may range from 1 to 3 lines, maybe Tail Truncation was clipping the string off to just one line?)